homeassistant/components/fritz/sensor.py

```diff
@@ -132,6 +162,42 @@ class SensorData(TypedDict, total=False):
         icon="mdi:download",
         state_provider=_retrieve_gb_received_state,
     ),
+    "line_kb_s_sent": SensorData(
+        name="Line capacity kb/s sent",
```

Comment:
```suggestion
        name="Line capacity kbit/s sent",
```

Reply:
In this parts are no changes by me.
Many of the values are given in byte instead of bit by the box/upstream library. 
I agree it should be in kbit/s to be consistent with usualy used nomenclature.

Reply:
`kb/s` = `kbit/s`, just different notation
What you are referring to is instead `kB/s`

Lower "**b**" is bits, upper "**B**" is bytes.

Simone

Reply:
Thank you for the explaination - since it wasn't my code originally i was not sure if it is correct...

BTW: I got a bunch of errors from the change form
unit_of_measurement="dB",
to
unit_of_measurement=SIGNAL_STRENGTH_DECIBELS,

is that unit of measurement not implemented? I am cautious to implement such a thing far upstream (realy new to home assistant)